Makefile

```diff
@@ -38,12 +38,30 @@ lint: lint.sub
 push: push.sub
 test: test.sub
 
+clean:
+	rm -f .dinit
+	rm -f .buildImage
+	docker rmi -f buildImage > /dev/null 2>&1 || echo -n ""
+
+docker-all: .buildImage
+	docker run --rm -ti \
+	  -v $(PWD):/go/src/github.com/kubernetes-incubator/service-catalog \
+	  buildimage make .dinit all
+
+.dinit: glide.yaml
```

Comment:
so the "d" was my way of saying this is used by the docker version of the build - so in this case its "docker's init file". I'll use a more descriptive name and add a comment.